iso.c

```diff
@@ -26,13 +26,14 @@ extern RD_BOOL g_encryption_initial;
 extern RDP_VERSION g_rdp_version;
 extern RD_BOOL g_use_password_as_pin;
 
-static RD_BOOL g_negotiate_rdp_protocol = True;
-
```

Comment:
I suggest that you don't change irrelevant parts in this PR.
If you feel that it should be fixed - make another PR.

Reply:
if g_rdp_version >= RDP_V5, we have to negotiate protocol to have the extended_data reply , so variable not needed.